src/app/routes/certRegRoutes.js

```diff
@@ -32,4 +38,36 @@ module.exports = function (app) {
         }
       }
     }))
-};
+
+    let courseId;
+    app.post(certRegServiceApi.reIssueCertificate,
+      bodyParser.json({ limit: '10mb' }),
+      permissionsHelper.checkPermission(),
+      proxy(certRegURL, {
+        proxyReqOptDecorator: proxyUtils.decorateRequestHeaders(),
+        proxyReqPathResolver: function (req) {
+          logger.debug({msg: `${certRegServiceApi.reIssueCertificate} is called with request: ${JSON.stringify(_.get(req, 'body'))}`});
```

Comment:
this should be logger.info method
"Certificate re-issue has called."

Dont add debug logs. In error log we should add request URL & payload as well.